src/router/routes/judging.ts

```diff
@@ -59,6 +59,14 @@ export class JudgingController extends ParentRouter implements IExpressControlle
             this.authService.verifyAcl(this.scoreAclPerm, AclOperations.CREATE),
             (req, res, next) => this.insertScoreHandler(req, res, next),
         );
+        app.post('/score',
```

Comment:
I would say to make these '/score/update' and '/score/delete' so that the routes don't interfere with the already existing POST route for '/score', which is for inserting scores.